action/container/container.go

```diff
@@ -25,10 +26,13 @@ var (
 // SetupOpts congregates options for Setup function
 // None of the values can be empty string, and mountContainerDir cannot be '.' or '/'
 type SetupOpts struct {
-	ContainerURL      string // URL or name of docker container
-	MountHostDir      string // Directory from host to mount into container
-	MountContainerDir string // Where to mount ^^^ host directory inside container
-	WorkdirContainer  string // Workdir of the container, specified by GITHUB_WORKSPACE environment variable
+	ContainerURL      string   // URL or name of docker container
+	MountHostDir      string   // Directory from host to mount into container
+	MountContainerDir string   // Where to mount ^^^ host directory inside container
+	WorkdirContainer  string   // Workdir of the container, specified by GITHUB_WORKSPACE environment variable
+	InputDir          string   // Directory for input files
```

Comment:
Overlooked this one here. Might be good to rename in the same notion to avoid confusion.

Reply:
Not sure what you mean. Is the current state unsatisfactory?

| Configuration option   | Direction            |
|:-----------------------|:--------------------:|
| RepoPath               | Host  --> Container  |
|                        |                      |
| ContainerOutputDirs    | Host <--  Container  |
| ContainerOutputFiles   | Host <--  Container  |
| OutputDir              | Host <--  Container  |
|                        |                      |
| InputDirs              | Host  --> Container  |
| InputFiles             | Host  --> Container  |
| ContainerInputDir      | Host  --> Container  |

Reply:
Sorry, I see what you mean now